peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -111,6 +111,23 @@ <h1 class="doc-title">
                 </button>
               </li>
             {% endif %}
+            {% block unconstitutional-provisions-tab %}
+              {% if perms.peachjam.view_unconstitutionalprovison and unconstitutional_provisions %}
```

Comment:
I wonder if a better way of doing this is to check for the perm in the view and not inject `unconstitutional_provisions` at all -- as it is, we'll still dump the JSON into the page even without perms.

Reply:
probably is, we should plan to change the other places that we do it in the templates, i'll leave as is for now and create an issue